src/components/dialog/dialog.js

```diff
@@ -221,13 +221,14 @@ class Dialog extends Modal {
 
   get closeIcon() {
     if (this.props.showCloseIcon) {
-      return <Icon
-        className="carbon-dialog__close"
+      return (<Icon
```

Comment:
I assume both will be valid. It just wanted you to add brackets as the code is over multiple lines?